WordPress/src/main/java/org/wordpress/android/ui/domainregister/suggestionslist/DomainSuggestionsFragment.kt

```diff
@@ -49,11 +51,15 @@ class DomainSuggestionsFragment : Fragment() {
             ToastUtils.showToast(activity, "Still under development.")
         }
         domainSearchEditText.addTextChangedListener(object : TextWatcher {
+            private val debouncer = Debouncer()
+
             override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {}
             override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {}
 
             override fun afterTextChanged(view: Editable?) {
-                viewModel.updateSearchQuery(view.toString())
+                debouncer.debounce(Void::class.java, {
```

Comment:
Just wondering whether we shouldn't cancel the request in the onCleared method? wdyt?